src/avram/tasks/db/drop.cr

```diff
@@ -3,8 +3,11 @@ require "colorize"
 class Db::Drop < BaseTask
   summary "Drop the database"
 
+  def initialize(@quiet : Bool = false)
+  end
+
   def run_task
-    Avram::Migrator::Runner.drop_db
+    Avram::Migrator::Runner.drop_db(@quiet)
     puts "Done dropping #{Avram::Migrator::Runner.db_name.colorize(:green)}"
```

Comment:
Sorry, I think this is the last thing 😅  This `puts` line should now go in to the drop_db method https://github.com/luckyframework/avram/blob/master/src/avram/migrator/runner.cr#L57 That way the `quiet?` option can be used to silence it like how the `create_db` does https://github.com/luckyframework/avram/blob/master/src/avram/migrator/runner.cr#L69-L71

Reply:
ah yes 👍 Looks like I didn't got it 😅 without that it wouldn't really be quiet 😁 Thanks for all the comments 💪 It gives me much better understanding how it is working 😃